kube_apiserver_metrics/datadog_checks/kube_apiserver_metrics/kube_apiserver_metrics.py

```diff
@@ -16,14 +16,26 @@
     'go_goroutines': 'go_goroutines',
     'APIServiceRegistrationController_depth': 'APIServiceRegistrationController_depth',
     'etcd_object_counts': 'etcd_object_counts',
+
+    # For Kubernetes < 1.14
     'rest_client_request_latency_seconds': 'rest_client_request_latency_seconds',
     'apiserver_admission_webhook_admission_latencies_seconds': 'admission_webhook_admission_latencies_seconds',
     'apiserver_admission_step_admission_latencies_seconds': 'admission_step_admission_latencies_seconds',
-    'apiserver_admission_controller_admission_duration_seconds': 'admission_controller_admission_duration_seconds',
+    'apiserver_admission_controller_admission_latencies_seconds': 'admission_controller_admission_duration_seconds',
```

Comment:
Was `apiserver_admission_controller_admission_duration_seconds` an inaccurate metric? If it's a rename, should we still keep the old one for older versions?

Reply:
The metric was named `apiserver_admission_controller_admission_latencies_seconds` in oldest versions of Kubernetes and it is now named `apiserver_admission_controller_admission_duration_seconds` in more recent versions.

In order to keep backward compatibility and in order to hide this change on Kubernetes side, both metrics are now mapped to the same Datadog metric: `admission_controller_admission_duration_seconds`.

So, there is no change in the set of metrics that this check can send.